root/static/scripts/edit/MB/TextList.js

```diff
@@ -35,11 +39,17 @@ MB.Form.TextList = function (input) {
     self.add = function (init_value) {
         $template.clone()
             .removeClass(template)
-            .insertAfter($template.parent().find('div.text-list-row').last())
+            .insertAfter($template
+                .parent()
+                .find('div.text-list-row')
+                .last())
             .show()
-            .find('input.value').attr("name", input + '.' + counter).val(init_value)
+            .find('input.value')
+                .attr("name", input + '.' + counter)
+                .val(init_value)
```

Comment:
I'm not really a fan of increasing the indentation here: it doesn't provide much in the way of readability and looks like a mistake to me. It's also inconsistent with almost all of our existing code, which keeps all methods at the same indentation level.

There are many other instances of this in the PR. I'd prefer if all the dots lined up and we didn't add multiple levels of indentation.